swagger/config.go

```diff
@@ -31,4 +33,6 @@ type Config struct {
 	PostBuildHandler PostBuildDeclarationMapFunc
 	// Swagger global info struct
 	Info Info
+	// [optional] If set, model builder should call this handler to get addition typename-to-swagger-format-field convertion.
+	AdditionalFormatHandler GetAdditionalSchemaFormat
```

Comment:
Its ok the change the current PR or squash it later

On Thu, Jun 23, 2016 at 10:49 AM, mbohlool <notifications@github.com> wrote:

> In swagger/config.go
> <https://github.com/emicklei/go-restful/pull/297#discussion_r68197246>:
>
> > @@ -31,4 +33,6 @@ type Config struct {
> >  	PostBuildHandler PostBuildDeclarationMapFunc
> >  	// Swagger global info struct
> >  	Info Info
> > +	// [optional] If set, model builder should call this handler to get addition typename-to-swagger-format-field convertion.
> > +	AdditionalFormatHandler GetAdditionalSchemaFormat
>
> You are right. the whole "additional" prefix was not a good idea. Fixed.
> Does it make sense to send new clean PR?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/emicklei/go-restful/pull/297/files/122305779cf3fc57bc9e0c9a28a38951c362b65c#r68197246>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAp4jO5uKIPaVANzC8ZYqe33pVKx8LX6ks5qOki3gaJpZM4I8P1o>
> .
>



-- 
Met vriendelijke groet,

^ Ernest Micklei

ernestmicklei.com